packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/overview.md

```diff
@@ -0,0 +1,3 @@
+Use the direct authentication OOB flow when you want to use an out-of-band factor as a primary factor. An out-of-band factor is a type of factor that requires a secondary verification method through a separate communication channel along with the initial user credentials.
+
+The steps in this guide focus on using the OOB flow with the Phone authenticator (SMS or Voice). Make sure that both **SMS** and **Voice call** are enabled.
```

Comment:
nit: **SMS** and/or **Voice call**. They don't both need to be enabled if only one is being used